app/models/user.rb

```diff
@@ -231,7 +231,7 @@ def send_reset_password_instructions
     end
   end
 
-  def never_used_account?
+  def not_invited_account?
```

Comment:
c'est pas pareil, en fait, non ? 
Là, c'est pas `not_invited`, c'est `maybe_invited_but_invitation_not_accepted_yet`.

Reply:
je sens le potentiel de dialogue de sourds :)

Pour moi, y'a 2 notions différentes, qui étaient implémentées n'importe comment avant : 

### Not invited
```ruby
scope :not_invited, -> { not_deleted.where(invitation_sent_at: nil) }
```
Là, c'est clair, pas de chichi, le champs `invitation_sent_at` est `nil`

### Invitation not accepted

Là, ça peut être des personnes déjà invitées, ou pas encore invitées, indifféremment. C'st pas pareil que `not invited`. Et on a plein de mail avec des particularismes destinés aux personnes `invitation_not_accepted`, genre "Vous n'avez pas encore accepté votre invitation", etc ; et pas forcément destiné aux personnes `not_invited`. Quand j'ai vu dans les nouveaux mailers qu'on affichait des trucs pour les gens `not_invited`, ça m'a fait tiquer.

Reply:
aah j'ai mal lu entre 'invitation_sent_at` et `invitation_accepted_at`. Pardon, oui c'est différent

Reply:
j'ai lu `invitation_sent_at` pour les deux

Reply:
J'ai scindé en deux commits et changé le nom pour `invitation_not_accepted`